src/parseTools.js

```diff
@@ -176,6 +176,7 @@ function needsQuoting(ident) {
 }
 
 globalThis.POINTER_SIZE = MEMORY64 ? 8 : 4;
+globalThis.POINTER_MAX = MEMORY64 ? 'Number.MAX_SAFE_INTEGER' : '2**32';
```

Comment:
is it ok that 2**32 doesn't match the number you had as startPointer before?

Reply:
Oops, that should have been `(2**32 - 1)` AKA `0xFFFFFFFF`.  For some reason the existing code used `(2**31 - 1)` AKA `0x&FFFFFFF`.. but that doesn't work for CAN_ADDRESS_2GB.

I have not idea why the existing code didn't use `0xFFFFFFFF`..

Reply:
But this fixes the tests under brower_2gb which were previously not working